flink-clients/src/main/java/org/apache/flink/client/ClientUtils.java

```diff
@@ -95,15 +100,55 @@ public static ClassLoader buildUserCodeClassLoader(List<URL> jars, List<URL> cla
 		return FlinkUserCodeClassLoaders.parentFirst(urls, parent);
 	}
 
+	public static JobSubmissionResult submitJob(
+		ClusterClient<?> client,
+		JobGraph jobGraph) throws ProgramInvocationException {
```

Comment:
Please indent the arguments by one more tab. Although not enforced everywhere, the arguments should be normally one more tab to the inside compared to the body of the method. This makes the code more readable.